common/http-api-client/src/lib.rs

```diff
@@ -14,6 +14,10 @@ use url::Url;
 
 pub use reqwest::IntoUrl;
 
+pub use user_agent::UserAgent;
+
+mod user_agent;
```

Comment:
shouldn't this be public?

Reply:
Currently not needed, so thought I'd leave it private for now